mmv1/third_party/terraform/tests/resource_vertex_ai_endpoint_test.go

```diff
@@ -0,0 +1,190 @@
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    Type: MMv1     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package google
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccVertexAIEndpoint_vertexAiEndpointNetwork(t *testing.T) {
+	t.Skip("Skip this test until b/254073292 is resolved")
```

Comment:
If we write a test without a network will it pass even with the quota issues? I'd prefer having *some* test rather than none even if it doesn't cover the network functionality

Reply:
It looks like the bug has been resolved, so I've removed the skip.